### PR TITLE
feat(route): 新增广州市人民政府

### PR DESCRIPTION
**Pull request by drgnchan**
## 完整路由地址 / Example for the proposed route(s)

```routes
/gov/gz/xw/gzyw
/gov/gz/xw/jrtt
/gov/gz/xw/tzgg
/gov/gz/zwgk/zcjd
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please place `gov`-related route inside `gov` folder

**Comment by drgnchan** [comment]
> Please place `gov`-related route inside `gov` folder

I have finished that.

File: lib/v2/gov/router.js [hunk 1]
```diff
@@ -51,4 +51,5 @@ module.exports = function (router) {
     router.get('/sichuan/deyang/govpulicinfo/:countyName', require('./sichuan/deyang/govpulicinfo'));
     router.get('/taiyuan/rsj/:caty/:page?', require('./taiyuan/rsj'));
     router.get('/xuzhou/hrss/:category?', require('./xuzhou/hrss'));
+    router.get('/gzgov/:channel/:category', require('./gzgov/index'));
```
Comment (TonyRL) [inline lib/v2/gov/router.js:54]: - The domain name is `gz` please use that instead.
- Please help maintain entries in this file in alphabetical order

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/gov/gz/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            const href = item.find('a').attr('href');
+            return {
+                title: item.find('a').attr('title'),
+                link: href,
+                pubDate: parseDate(item.find('span').text()),
+            };
+        });
+
+    ctx.state.data = {
+        title: `广州市人民政府 - ${categoryMap[category]}`,
```
Comment (TonyRL) [inline lib/v2/gov/gz/index.js:41]: You can extract the page title using
```suggestion
        title: `广州市人民政府 - ${$('.main_title').text()}`,
```
In this way, there's no need to update the entries in `categoryMap` above.

File: lib/v2/gov/gz/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+const rootUrl = 'http://gz.gov.cn';
+
+const categoryMap = {
+    gzyw: '广州要闻',
+    jrtt: '今日头条',
+    tzgg: '通知公告',
+    zcjd: '政策解读',
+};
+
```
Comment (TonyRL) [inline lib/v2/gov/gz/index.js:13]:
```suggestion
```

**Comment by drgnchan** [comment]
@TonyRL Thanks a lot.All done.

**Comment by drgnchan** [comment]
@TonyRL hi,can you review if you have time to?

File: docs/government.md [hunk 4]
```diff
@@ -180,6 +180,28 @@ pageClass: routes
 
 <Route author="Fatpandac" example="/gov/huizhou/zwgk/jgdt" path="/gov/huizhou/zwgk/:category?" :paramsDesc="['资讯类别，可以从网址中得到，默认为政务要闻']"/>
 
+## 广州市人民政府
+
+### 新闻
+
+<Route author="drgnchan" example="/gov/gz/xw/gzyw" path="/gov/gz/xw/:category">
```
Comment (TonyRL) [inline docs/government.md:187]: Missing `:paramsDesc` attributes

File: docs/government.md [hunk 5]
```diff
@@ -180,6 +180,28 @@ pageClass: routes
...
+
+| 广州要闻 | 今日头条 | 通知公告 |
+| ---- | ---- | ---- |
+| gzyw | jrtt | tzgg |
+
+</Route>
+
+### 政务公开
+
+<Route author="drgnchan" example="/gov/gz/zwgk/zcjd" path="/gov/gz/zwgk/:category">
```
Comment (TonyRL) [inline docs/government.md:197]: Missing `:paramsDesc` attributes

**Review by TonyRL — COMMENTED** [review]
Missing respective updates for `maintainer.js` and `radar.js`

**Pull request merged**